packages/google-maps/README.md

```diff
@@ -313,6 +313,7 @@ The following properties are available for adjusting the camera view on initiali
 `zoom` | `number` | Zoom level (described [here](https://developers.google.com/maps/documentation/javascript/tutorial#zoom-levels))
 `bearing` | `number` | Bearing, in degrees
 `tilt` | `number` | Tilt, in degrees
+`disableMarkerTapHandler` | `boolean` | Disables the default marker event handling (panning/info windows) done by Google Maps
```

Comment:
I think a better name would be `preventDefaultMarkerTapBehavior` or `disableDefaultMarkerTapBehavior` or similar. To indicate that we want to disable the default behavior rather than the handler itself.

Reply:
Good call. Made those changes.